cmd/sonarExecuteScan_test.go

```diff
@@ -253,7 +253,7 @@ func TestSonarLoadScanner(t *testing.T) {
 		// assert
 		assert.NoError(t, err)
 		assert.Equal(t, url, mockClient.requestedURL[0])
-		assert.Regexp(t, "sonar-scanner-cli-4.3.0.2102-linux.zip$", mockClient.requestedFile[0])
+		assert.Regexp(t, "sonar-scanner-cli-4.4.0.2170-linux.zip$", mockClient.requestedFile[0])
```

Comment:
I guess it would make sense to replace the actual version here with a wildcard.

Reply:
How would this look like?

Reply:
As this is a regex, maybe like this.

```suggestion
		assert.Regexp(t, "sonar-scanner-cli-[0-9\.]*-linux\.zip$", mockClient.requestedFile[0])
```

Reply:
This causes failures on `go run pkg/generator/step-metadata.go`

> Error occured: exit status 1
> exit status 1

Reply:
Ok, then let's merge this for now.